final/final.go

```diff
@@ -415,13 +427,14 @@ func eurozone(countries []common.Country) common.Region {
 	countryCodes := findCountriesByCurrency(countries, "EUR")
 	theseCountries := findCountries(countries, countryCodes)
 	return common.Region{
-		Id:                 "eurozone",
-		Name:               "Eurozone",
-		Countries:          countryCodes,
-		Currencies:         currenciesForCountries(theseCountries),
-		Languages:          languagesForCountries(theseCountries),
-		MeasurementSystems: measurementSystemsForCountries(theseCountries),
-		Timezones:          timezonesForCountries(theseCountries),
+		Id:                   "eurozone",
+		Name:                 "Eurozone",
+		Countries:            countryCodes,
+		Currencies:           currenciesForCountries(theseCountries),
+		Languages:            languagesForCountries(theseCountries),
+		MeasurementSystems:   measurementSystemsForCountries(theseCountries),
+		Timezones:            timezonesForCountries(theseCountries),
+		DefaultDeliveredDuty: theseCountries[0].DefaultDeliveredDuty,
```

Comment:
not sure we need default delivered duty on region... in our use case each experience will have a country assigned to it that is the default -think we should just use that one.